query/sources/impl/src/main/java/org/codice/ddf/admin/sources/csw/CswSourceUtils.java

```diff
@@ -96,10 +97,14 @@ public CswSourceUtils(RequestUtils requestUtils, SourceUtilCommons sourceUtilCom
      */
     public ReportWithResult<ResponseField> discoverCswUrl(HostField hostField,
             CredentialsField creds) {
-        return requestUtils.discoverUrlFromHost(hostField,
+        ReportWithResultImpl<ResponseField> responseResult = requestUtils.discoverUrlFromHost(hostField,
                 URL_FORMATS,
                 creds,
                 GET_CAPABILITIES_PARAMS);
+        if(responseResult.containsErrorMsgs()) {
```

Comment:
I'd prefer we don't do this, it's pretty much ignoring all the messages from the discoverUrlFormHost